Source/WebCore/Modules/permissions/PermissionStatus.cpp

```diff
@@ -101,4 +150,32 @@ void PermissionStatus::eventListenersDidChange()
     m_hasChangeEventListener = hasEventListeners(eventNames().changeEvent);
 }
 
+MainThreadPermissionObserver::MainThreadPermissionObserver(WeakPtr<PermissionObserver> workerPermissionObserver, ScriptExecutionContextIdentifier workerContextIdentifier, const PermissionDescriptor& workerDescriptor, const ClientOrigin& workerOrigin, PermissionQuerySource source, Page* page, PermissionState state)
+    : m_identifier(nextMainThreadPermissionObserverIdentifier())
```

Comment:
So MainThreadPermissionObserver::m_identifier doesn't match PermissionStatus::m_mainThreadPermissionObserverIdentifier ?

Reply:
Oh this is an issue. I'll fix this.